bases/renku_data_services/data_api/app.py

```diff
@@ -134,6 +135,24 @@ def register_all_handlers(app: Sanic, config: Config) -> Sanic:
         authenticator=config.authenticator,
         internal_gitlab_authenticator=config.gitlab_authenticator,
     )
+    notebooks = NotebooksBP(
+        name="notebooks_old",
+        url_prefix=url_prefix,
+        authenticator=config.authenticator,
+        nb_config=config.nb_config,
+        internal_gitlab_authenticator=config.gitlab_authenticator,
+        git_repo=config.git_repositories_repo,
+    )
+    notebooks_new = NotebooksNewBP(
```

Comment:
refactor suggestion: rename to sessions.
Even if the session can start notebooks as the old implementation, it will make the code clearer.
I would suggest adding a task to #303 about that.

Reply:
I will just add a task for this in #303 I dont want to rename this now because it may cause additional conflicts in the many PRs that branch off of this.

Reply:
Sure thing, that's why I suggested to add it to #303.
All good from my side.